### ISSUE TITLE
Heroku Setup with yabeda-prometheus

### ISSUE THREAD

**Issue by jonatasrancan** [yabeda-rb/yabeda-sidekiq#35]
Hi there, before anything very nice 😃 

Our app runs on heroku, we manage to setup `yabeda-sidekiq` with a custom exporter we built to send metrics to the logs, so we could parse it somewhere else, worked without problem

But I was trying to use `yabeda-prometheus` and the `/metric` endpoint. Locally I got the same scenario from here https://github.com/yabeda-rb/yabeda-sidekiq/issues/27, tried configuring both suggested solutions but I couldn't get all metrincs in a single `/metric` endpoint.

But reading all the info about multi process and containerize scenarios, I come to the conclusion that the combo `yabed-sidekiq`+ `yabeda-prometheus` won't work on a heroku setup that have sidekiq running in multiple dynos, correct? I can't expose everything and consolidate the data in a safe way before exposing it.

Or did I miss something?

**Comment by sobrinho** [comment]
You are correct there. Dynos does not share memory/filesystem at all, they are fully isolated.

The best you can try here is to build a custom exporter that would use a unified database such as Redis/Memcached and then you could have a single /metrics endpoint.

For Kubernetes for instance you usually scrape the different kind of pods to gather the full picture. I would assume you need the same in Heroku by retrieving the logs from every dyno.

**Comment by sobrinho** [comment]
@Envek you can close this one.
